specs/Accessibility.md

```diff
@@ -0,0 +1,72 @@
+# Requirements
+
+## ARIA conformance
+In general, components need to conform to the [WAI-ARIA Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/). These practices are evaluated to achieve best possible user experience with supported browser/screen reader combinations.
+
+## Keyboard navigation
+- All elements that are interactive need to be focusable using keyboard keys (TAB/Shift+Tab or arrow keys)
+- Focused element needs to be highlighted if it was focused by keyboard
+- Focus should never go to the ``<body>`` element
+- User needs to be able to move focus at any time, focus should not be be 'stuck' on one element
+- In RTL mode, left arrow key moves the focus to the next focusable element, right arrow key moves the focus to the previous focusable element within the focus zone.
+
+## Focus
+- Whenever possible, focus should land only on elements that have concrete implicit or explicit ARIA role (`<button />`, `role='button'`, `role='menuitem'`, ....)
+- onClick event handlers need to be handled on focusable elements with implicit or explicit ARIA role and not on their parent/child elements
+- When reacting on focus or on hover screen reader interaction needs to be specified. Some screen readers do not have concept of hovering and not always focus elements, so there needs to be another way for the screen reader users to interact with such elements. For example showing popups on focus or on hover needs to be justified and alternative interaction compatible with the screen readers need to be provided
+
+## Screen reader support
+User needs to be able to interact with the application using:
+- screen reader with virtual cursor navigation
+- screen reader witout virtual cursor navigation
+
+All requirements for keyboard navigation are valid in both of these screen reader use cases. User needs to be able to understand which element is focused and how he can interact with it just based on the information provided by the screen reader.
+
+Following combinations of clients and screen readers is supported/required:
+- Windows/Chrome - JAWS, NVDA
+- Windows/Edge - Narrator
```

Comment:
I think this is valid only for old Edge, not for Edge Chromium.
I believe Edge Chromium should work with JAWS and NVDA

Reply:
all combinations _should_ work, this is just saying which of those we test and support officially.